.github/workflows/ruby.yml

```diff
@@ -0,0 +1,27 @@
+name: Ruby
+
+on:
+  push:
+  pull_request:
```

Comment:
GitHub Actions presents a configuration that targets master branch by default, but I followed @tubbo's comment <https://github.com/redis-store/redis-actionpack/pull/39#discussion_r787106226> to make it work on all branches.